nannou/src/wgpu/texture/image.rs

```diff
@@ -588,7 +588,7 @@ where
         .extent(wgpu::Extent3d {
             width,
             height,
-            depth: array_layers,
+            depth_or_array_layers: array_layers,
```

Comment:
I'm glad they clarified this - I remember asking in the wgpu users room if depth applied to the layer count :laughing: